### PR TITLE
Handle PipelineRun that failed to start

### PR DESCRIPTION
**Pull request by jessm12**
# Changes

When a PipelineRun fails to start due to an error, for example it has a missing PipelineResource, render a message to say that the details could not be retrieved due to an error and display this error on the PipelineRun details panel as opposed to the current behaviour of displaying a blank screen
# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/dashboard/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

**Comment by jessm12** [comment]
Thanks @AlanGreene 😄 something similar to this is what I had wanted to do but wasn't sure how to go about it! I will work from your code now

**Comment by jessm12** [comment]
please take a look @AlanGreene 🙂 

Screenshot 2019-06-06 at 11 22 55

File: src/containers/PipelineRun/PipelineRun.js [hunk 1]
```diff
@@ -165,7 +164,36 @@ export /* istanbul ignore next */ class PipelineRunContainer extends Component {
...
+          <RunHeader
+            lastTransitionTime={lastTransitionTime}
+            loading={loading}
+            runName={pipelineRunName}
+            reason="Error"
+            status={pipelineRunStatus}
+          />
+          <InlineNotification
+            kind="error"
+            title="Unable to retrieve PipelineRun logs"
```
Comment (AlanGreene) [inline src/containers/PipelineRun/PipelineRun.js]: This title is a little misleading, the actual error described is that the PipelineRun couldn't be created.

Maybe we should go with something more generic like 'Unable to load PipelineRun details' as there are likely other scenarios that can result in the same state. We could also include `pipelineRunError.reason` for additional info.

Comment (jessm12) [reply src/containers/PipelineRun/PipelineRun.js]: Sounds good I will make that change 👍

**Pull request merged**